types/slickgrid/index.d.ts

```diff
@@ -3,6 +3,7 @@
 // Definitions by: Josh Baldwin <https://github.com/jbaldwin>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
+// @ts-nocheck
```

Comment:
This d.ts file isn't ready for ts 4.9 (hopefully this file will be removed soon from the repo)